dark/index.html

```diff
@@ -244,47 +295,5 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
 
   <!-- Internal JavaScript -->
   
-  <script>
-    // Auto Typing Effect Script
-
-    var typed = new Typed(".auto-type", {
-      strings: [
-        "STUDENT",
-        "DEVELOPER",
-        "PROGRAMMER",
-        "DESIGNER",
-        "EDITOR",
-        "PHOTOGRAPHER",
-        "FREELANCER",
-      ],
-      typeSpeed: 100,
-      backSpeed: 100,
-      loop: true,
-    });
-
-    
-    // Navigation Bar Scroll Animation
-    
-    $(window).scroll(function () {
-      if ($(document).scrollTop() > 400) {
-        $(".nav-bar").addClass("affix");
-        console.log("OK");
-      } else {
-        $(".nav-bar").removeClass("affix");
-      }
-    });
-
-    
-    // Mobile View Navigation Bar Menu Icon
-    
-    const menu_toggle = document.querySelector(".menu-toggle");
-    const sidebar = document.querySelector(".sidebar");
-
-    menu_toggle.addEventListener("click", () => {
-      menu_toggle.classList.toggle("is-active");
-      sidebar.classList.toggle("is-active");
-    });
-  </script>
-</body>

```

Comment:
Don't remove this body ending tag, keep them like earlier @Neutx

Reply:
Again thanks for reminding the body ending tag was also misplaced.

Reply:
Thanks for solving them @Neutx